### ISSUE TITLE
Stats do not change

### ISSUE THREAD

**Issue by Qiangong2** [HKOServer/HKO-re#40]
As of right now, mobs do not do any damage, but the damage animation shows up when fighting. This needs to be fixed.

**Comment by Qiangong2** [comment]
Expanding this issue as @Redcrafter requested. 

Stats as a whole are currently broken, you don't gain or lose hp and your specific abilities (farming, crafting, etc.) do not cost stamina.

**Comment by Qiangong2** [comment]
Hey @PyroSamurai, just wanted to see if you'd be interested in giving your input on this? The abilities work (as you can farm, craft, mine, etc.) and you can damage enemies, but there's no penalty for it at all. If you need clarification on the code, let me or @Redcrafter know and we can help out

**Comment by PyroSamurai** [comment]
@Qiangong2 Can you point me to the relevant code files?

**Comment by Qiangong2** [comment]
@PyroSamurai https://github.com/HelloKittyOnline/HKO-re/blob/main/Server/Protocols/Battle.cs holds all the battle/mob data and everything that starts with `client.StartAction` should consumer stamina

**Comment by PyroSamurai** [comment]
Correct me if this is intentional, but it doesn't seem like you are touching the players stats at all on the server-side.
```
// TODO: implement player hp and stamina
// client.Player.Hp -= playerDamage;
```
Anyways, I apologize if this is all wrong since I'm unable to compile the server and run a full setup b/c I don't use Windows. So this is all just manual code review.

**Comment by Redcrafter** [comment]
Yes it's not implemented because we don't know the damage formula for the game.
Btw you should be able to compile the server just fine on linux.

**Comment by PyroSamurai** [comment]
@Redcrafter Then how is this a bug? It's simply a TODO. 
"compile the server just fine on linux" I hope it continues to do so. However without MariaDB and client installed I couldn't really test anything anyways, and installing those is a bit much for getting a second opinion on a bug.
